From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Grische <2787581+grische@users.noreply.github.com>
Date: Fri, 10 Apr 2026 21:41:39 +0200
Subject: [PATCH 9/9] fix: include sysinfo.h to fix modern musl builds

This impacts OpenWrt 25.12 and later versions.

(cherry picked from commit aa409ee39fbc34465c03ef1444321cecef27646a)
---
 src/information.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/information.c b/src/information.c
index 3a82164..6089c25 100644
--- a/src/information.c
+++ b/src/information.c
@@ -7,8 +7,7 @@
 #include <errno.h>
 
 #include <arpa/inet.h>
-#include <linux/unistd.h>
-#include <linux/kernel.h>
+#include <sys/sysinfo.h>
 
 #include "batadv.h"
 #include "util.h"
-- 
2.53.0

